docs/edc/EDC_BPDM_Setup.postman_collection.json

```diff
@@ -1,1391 +1,1524 @@
-    },
-    {
-      "key": "ASSET_GET_BPL_INPUT",
-      "value": "GET_BPL_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_PUT_BPL_INPUT",
-      "value": "PUT_BPL_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPL_INPUT_SEARCH",
-      "value": "POST_BPL_INPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_BPS_INPUT",
-      "value": "GET_BPS_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_PUT_BPS_INPUT",
-      "value": "PUT_BPS_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPS_INPUT_SEARCH",
-      "value": "POST_BPS_INPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_BPA_INPUT",
-      "value": "GET_BPA_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_PUT_BPA_INPUT",
-      "value": "PUT_BPA_INPUT",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPA_INPUT_SEARCH",
-      "value": "POST_BPA_INPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPL_OUTPUT_SEARCH",
-      "value": "POST_BPL_OUTPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPS_OUTPUT_SEARCH",
-      "value": "POST_BPS_OUTPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPA_OUTPUT_SEARCH",
-      "value": "POST_BPA_OUTPUT_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_INPUT_CHANGELOG_SEARCH",
-      "value": "POST_INPUT_CHANGELOG_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_OUTPUT_CHANGELOG_SEARCH",
-      "value": "POST_OUTPUT_CHANGELOG_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_LEGAL_FORMS",
-      "value": "GET_LEGAL_FORMS",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_IDENTIFIER_TYPES",
-      "value": "GET_IDENTIFIER_TYPES",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_FIELD_QUALITY_RULES",
-      "value": "GET_FIELD_QUALITY_RULES",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_BPL_POOL",
-      "value": "GET_BPL_POOL",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPL_POOL_SEARCH",
-      "value": "POST_BPL_POOL_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_BPS_POOL",
-      "value": "GET_BPS_POOL",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPS_POOL_SEARCH",
-      "value": "POST_BPS_POOL_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_GET_BPA_POOL",
-      "value": "GET_BPA_POOL",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_BPA_POOL_SEARCH",
-      "value": "POST_BPA_POOL_SEARCH",
-      "type": "string"
-    },
-    {
-      "key": "ASSET_POST_CHANGELOG_POOL_SEARCH",
-      "value": "POST_CHANGELOG_POOL_SEARCH",
-      "type": "string"
-    }
-  ]
+	"info": {
+		"_postman_id": "aab9dc22-2777-4429-8e0e-e72311c54d6b",
+		"name": "EDC BPDM Setup Template",
```

Comment:
Can we change the name, EDC BPDM Setup Template -> EDC BPDM Setup

Reply:
Changed the name back to the original